x-pack/legacy/plugins/reporting/export_types/common/lib/screenshots.js

```diff
@@ -287,29 +289,34 @@ export function screenshotsObservableFactory(server) {
           tap(() => logger.debug('setting viewport')),
           mergeMap(
             ({ browser, itemsCount }) => setViewport(browser, itemsCount, layout),
-            ({ browser, itemsCount }) => ({ browser, itemsCount }),
+            ({ browser, itemsCount }) => ({ browser, itemsCount })
           ),
           tap(({ itemsCount }) => logger.debug(`waiting for ${itemsCount} to be in the DOM`)),
           mergeMap(
             ({ browser, itemsCount }) => waitForElementsToBeInDOM(browser, itemsCount, layout),
             ({ browser, itemsCount }) => ({ browser, itemsCount })
           ),
-          tap(() => logger.debug('positioning elements')),
+          // Waiting till _after_ elements have rendered before injecting our CSS
```

Comment:
This is adding a change from another PR which missed getting backported to 7.x: https://github.com/elastic/kibana/pull/31949#issuecomment-513022111